docs/getting-started/create-local-module.md

```diff
@@ -98,7 +98,7 @@ export const register: ModuleRegisterFunction<FireflyRuntime, AppContext> = (run
 
     runtime.registerNavigationItem({
         $label: "Local/Page",
-        to: "/local/page"
+        $to: "/local/page"
```

Comment:
Couldn't we have kept the field name as is for backwards compatibility?

Reply:
Too complicated, it’s not easy to play with the nav items.

It would have required to duplicated interfaces and duplicated the render functions for the consumers.